JavaScript/signin.js

```diff
@@ -0,0 +1,67 @@
+
+const registeredEmails = {
```

Comment:
등록된 계정이 나타날 수 있게 변수명을 정해주시면 좋을 것 같아요!

```suggestion
const registeredAccount = {
```

Reply:
email, password가 한 쌍이니 아래와 같은 구조가 여러 개정이 있을 때는 더 적절할 것 같아요.

```
const registeredAccount = [
  { email:  'test@codeit.com', password: 'codeit101' },
  { email:  'guest@codeit.com', password: 'codeit101' },
]
```